app.rb

```diff
@@ -0,0 +1,163 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+class App
+  attr_reader :people
+
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def menu
+    puts ''
+    puts 'Please choose an option by entering a number:'
+    puts '1 - List all books 📚'
+    puts '2 - List all people 🧟‍♂️🧟'
+    puts '3 - Create a person 👶'
+    puts '4 - Create a book 📗'
+    puts '5 - Create a rental 🗝'
+    puts '6 -  List all rentals for a given person id 🗝🗝'
+    puts '7 - Exit 🚪'
+  end
+
+  def option_checker(answer)
+    case answer
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_rentals
+    else
+      puts '⚠️ Wrong input!'
+      run
+    end
```

Comment:
- Please note that when I type `7` in the app a get a `wrong input` instead of exiting the app. Kindly add the code to exit the app example
```ruby
  when "7"
      # *Have a way to quit the app.
      puts "Thank you for using this app!"
    end
```

Reply:
Hello @rotimiazeez. Thank you for your kind feedback.
If I change it to the way to mentioned. I am getting linter errors like below. I can not have more than 7 cases. What do you recommend disabling linters or not changing anything?
`Cyclomatic complexity for option_checker is too high`

Reply:
<img width="1552" alt="Screen Shot 2022-08-05 at 11 31 20" src="https://user-images.githubusercontent.com/25887253/183036926-d6da0783-8ec9-42f9-acfc-8361436bea13.png">

Reply:
@huseyinbiyik can you try disabling the linter for that line? 🤔
try using 
```
# rubocop:disable Metrics/CyclomaticComplexity
.... your method here
# rubocop:enable Metrics/CyclomaticComplexity
```

Reply:
Sure. I disabled linters, thank you!